torch/nn/parallel/distributed.py

```diff
@@ -316,6 +316,15 @@ class DistributedDataParallel(Module):
                          are getting different gradients, which should not
                          happen if DistributedDataParallel is correctly used.
                          (default: ``False``)
+        grad_is_view: when setting to ``True``, grads will be views of all reduce
+                      buckets in reducer, this can reduce peak memory usage, the
+                      saved memory size will be equal to the total grads size.
+                      Also it can avoid coping between grads and the all reduce
+                      buckets in reducer and thus improve training performance.
+                      When grads are views, detach_() can not be called in your
+                      zero_grad() function, please fix it by referring to
+                      the Optimizer.zero_grad() function in torch/optim/optimizer.py
```

Comment:
double quote all code. 

BTW, can we make `zero_grad()` a link by using `:meth:torch.optim.Optimizer.zero_grad`?